core/scss/components/main-nav/_main-nav.scss

```diff
@@ -0,0 +1,334 @@
+@charset "UTF-8";
+
+//
+// Main Navigation
+//
+// The primary site navigation that is usually in the header region of a website.
+// It is a horizontal menu bar on larger breakpoints, while on smaller breakpoints,
+// it changes into a vertical menu and the display is toggled by a button.
+//
+// This component is implemented to be accessible. For more information, please see:
+//
+// https://github.com/SU-SWS/decanter/wiki/About-the-Main-Nav-Component
+//
+// .su-main-nav--light          - light colored theme
+// .su-main-nav--mobile-search  - include search form inside the mobile nav
+// .su-main-nav--center         - center the nav
+// .su-main-nav--right          - right justify the nav
+//
+// Markup: ../templates/components/main-nav/main-nav.twig
+//
+// Style guide: Components.Main Navigation
+//
+.su-main-nav {
+  @include font-smoothing;
+  font-family: $font-sans;
+  position: relative;
+  z-index: 9999;
+
+  ul {
+    @include margin(0 null);
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    list-style: none;
+    background-color: $color-black;
+  }
+
+  //top level styles
+  > ul {
+    > li {
+      @include margin(null null 0);
+
+      > a {
+        font-weight: 600;
+      }
+    }
+
+    @include grid-media('lg') {
+      @include padding(null null null 0);
+      flex-direction: row;
+      flex-wrap: wrap;
+      background-color: transparent;
+
+      > li {
+        > a {
+          @include fancy-hover;
+          @include padding(null 0);
+          @include margin(0 1.16em 0 0);
+          font-size: 1.05em;
+          border-bottom: 0;
+
+          &:active {
+            font-weight: 600;
```

Comment:
Don't we have a font-weight variable for this? Yeah, `$font-bold` is 600. Please replace with core variable.